### PR TITLE
Feature/nijo 616 prepare for prod - part1

### PR DESCRIPTION
**Pull request by mingazhev**
1. Fixed race possibility
2. Added sentry

### CODE REVIEW

File: HappyTravel.Edo.PaymentProcessings/HappyTravel.Edo.PaymentProcessings.csproj [hunk 1]
```diff
@@ -3,13 +3,15 @@
...
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="HappyTravel.VaultClient" Version="1.0.5" />
-    <PackageReference Include="IdentityModel" Version="4.0.0" />
+    <PackageReference Include="IdentityModel" Version="4.3.0" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
-    <PackageReference Include="Sentry.AspNetCore" Version="1.2.0" />
+    <PackageReference Include="Sentry.AspNetCore" Version="2.1.1" />
+    <PackageReference Include="HappyTravel.StdOutLogger" Version="1.0.11" />
```
Comment (kirillta) [inline HappyTravel.Edo.PaymentProcessings/HappyTravel.Edo.PaymentProcessings.csproj:14]: The latest version is 1.0.14

File: Helm_v3/pp/values_prod.yaml [hunk 2]
```diff
@@ -62,4 +62,5 @@ affinity: {}
 
 env:
   ASPNETCORE_ENVIRONMENT: "Development"
```
Comment (kirillta) [inline Helm_v3/pp/values_prod.yaml:64]: Must be Production and Staging as well. @gans13

Comment (gans13) [reply Helm_v3/pp/values_prod.yaml:64]: Yes

File: HappyTravel.Edo.PaymentProcessings/HappyTravel.Edo.PaymentProcessings.csproj (hunk 1 above)
Comment (mingazhev) [reply HappyTravel.Edo.PaymentProcessings/HappyTravel.Edo.PaymentProcessings.csproj]: Fixed

File: Helm_v3/pp/values_prod.yaml (hunk 2 above)
Comment (mingazhev) [reply Helm_v3/pp/values_prod.yaml]: Fixed

**Pull request merged**